<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvPacsConnection</class>
 <widget class="QWidget" name="vvPacsConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>802</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_ID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient ID</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="patientID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="scanButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SCAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="patientName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_NAME">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient name</string>
   </property>
  </widget>
  <widget class="QToolButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">...</string>
   </property>
   <property name="icon">
    <iconset resource="../vvIcons.qrc">
     <normaloff>:/common/icons/basket_download.png</normaloff>:/common/icons/basket_download.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Patients</string>
   </property>
   <widget class="QTreeView" name="patientTreeView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Studies</string>
   </property>
   <widget class="QTreeView" name="studyTreeView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTreeView" name="seriesTreeView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="imagesTreeView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>321</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Series</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>451</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Images</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_PHYS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_PHYS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Physician Ref.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SDESC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Study Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_SDESC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabFilter">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="modalityTab">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Modalities</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>264</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="check_MR">
          <property name="text">
           <string>MR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_CR">
          <property name="text">
           <string>CR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_OT">
          <property name="text">
           <string>OT</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_RF">
          <property name="text">
           <string>RF</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_SC">
          <property name="text">
           <string>SC</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_CT">
          <property name="text">
           <string>CT</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="check_US">
          <property name="text">
           <string>US</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_NM">
          <property name="text">
           <string>NM</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_DR">
          <property name="text">
           <string>DR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_SR">
          <property name="text">
           <string>SR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_XA">
          <property name="text">
           <string>XA</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_MG">
          <property name="text">
           <string>MG</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="check_ModAll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>All</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="dateTab">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Date</string>
    </attribute>
    <widget class="QComboBox" name="comboDate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Before</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>After</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Between</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateBetweenEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabNetwork">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="networkSelect">
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QComboBox" name="networkCombo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="networkConfig">
    <attribute name="title">
     <string>Network management</string>
    </attribute>
    <widget class="QPushButton" name="NetworkButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>179</width>
       <height>120</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ID_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="AETitleEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_ID_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Adress</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="AdressEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_ID_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="PortEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ID_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>AE Title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="removeNetworkButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_help">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">...</string>
   </property>
   <property name="icon">
    <iconset resource="../vvIcons.qrc">
     <normaloff>:/common/icons/bullet_info.png</normaloff>:/common/icons/bullet_info.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="optionsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="../vvIcons.qrc">
     <normaloff>:/common/icons/edit.png</normaloff>:/common/icons/edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_email_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QToolButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">...</string>
   </property>
   <property name="icon">
    <iconset resource="../vvIcons.qrc">
     <normaloff>:/common/icons/standardbutton-cancel-16.png</normaloff>:/common/icons/standardbutton-cancel-16.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_exit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>label_ID</zorder>
  <zorder>patientID</zorder>
  <zorder>scanButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>patientName</zorder>
  <zorder>label_NAME</zorder>
  <zorder>importButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>seriesTreeView</zorder>
  <zorder>imagesTreeView</zorder>
  <zorder>text_PHYS</zorder>
  <zorder>label_PHYS</zorder>
  <zorder>label_SDESC</zorder>
  <zorder>text_SDESC</zorder>
  <zorder>tabFilter</zorder>
  <zorder>tabNetwork</zorder>
  <zorder>label_2</zorder>
  <zorder>label_help</zorder>
  <zorder>helpButton</zorder>
  <zorder>optionsButton</zorder>
  <zorder>label_email_2</zorder>
  <zorder>exitButton</zorder>
  <zorder>label_exit</zorder>
 </widget>
 <tabstops>
  <tabstop>scanButton</tabstop>
  <tabstop>patientID</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>patientName</tabstop>
  <tabstop>importButton</tabstop>
  <tabstop>seriesTreeView</tabstop>
  <tabstop>imagesTreeView</tabstop>
  <tabstop>text_PHYS</tabstop>
  <tabstop>text_SDESC</tabstop>
  <tabstop>tabFilter</tabstop>
  <tabstop>tabNetwork</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>optionsButton</tabstop>
  <tabstop>patientTreeView</tabstop>
  <tabstop>studyTreeView</tabstop>
  <tabstop>check_MR</tabstop>
  <tabstop>check_CR</tabstop>
  <tabstop>check_OT</tabstop>
  <tabstop>check_RF</tabstop>
  <tabstop>check_SC</tabstop>
  <tabstop>check_CT</tabstop>
  <tabstop>check_US</tabstop>
  <tabstop>check_NM</tabstop>
  <tabstop>check_DR</tabstop>
  <tabstop>check_SR</tabstop>
  <tabstop>check_XA</tabstop>
  <tabstop>check_MG</tabstop>
  <tabstop>check_ModAll</tabstop>
  <tabstop>comboDate</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>dateBetweenEdit</tabstop>
  <tabstop>networkCombo</tabstop>
  <tabstop>NetworkButton</tabstop>
  <tabstop>NameEdit</tabstop>
  <tabstop>AETitleEdit</tabstop>
  <tabstop>AdressEdit</tabstop>
  <tabstop>PortEdit</tabstop>
  <tabstop>removeNetworkButton</tabstop>
 </tabstops>
 <resources>
  <include location="../vvIcons.qrc"/>
 </resources>
 <connections/>
</ui>
